src/Sparrow/Json/AbstractBlittableJsonTextWriter.cs

```diff
@@ -3,11 +3,12 @@
 using System.Globalization;
 using System.IO;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
 using Sparrow.Extensions;
 
 namespace Sparrow.Json
 {
-    public abstract unsafe class AbstractBlittableJsonTextWriter
+    public abstract unsafe class AbstractBlittableJsonTextWriter : IBlittableJsonTextWriter
```

Comment:
Any callers that accept this directly? If we dropped the virtual call, won't we miss something here?

Reply:
There are a few that will accept `AbstractBlittableJsonTextWriter` but as it was, it has almost no virtual calls with the exception of `.Flush()` which we dont care to be virtual at all. It's not a big deal, but most likely it would make sense to do a more complete revam of the Blittable JSON Text Writer anyways. I just went to straight fix the issue for now.